x-pack/solutions/security/plugins/security_solution/public/rules/links.ts

```diff
@@ -53,23 +53,21 @@ export const links: LinkItem = {
           defaultMessage: 'SIEM Rules',
         }),
       ],
-      capabilities: `${SECURITY_FEATURE_ID}.show`,
+      capabilities: [[`${SECURITY_FEATURE_ID}.show`, `${SECURITY_FEATURE_ID}.detections`]],
```

Comment:
@tomsonpl I've been told that you decided the rules page should only be available with `detections` capability, and it should be hidden entirely with `external_detections` (AI4SOC product), is this correct?

Reply:
@sergi hmm, That was the initial assumption, however later we learned that the rule details page should be available in external detections. Best to check with @xcrzx what the current status is, because I think that has been reverted too :p

Reply:
@xcrzx I assume, based on [this](https://github.com/elastic/kibana/pull/219930#issuecomment-2862456410), that none of the regular detection rules page should be available in the AI4SOC product.

Reply:
I'll let @xcrzx confirm, but to my understanding that is correct yes